### ISSUE TITLE
Question about clip_cov implementation and comparison with kl_cov

### ISSUE THREAD

**Issue by curryandsun** [PRIME-RL/Entropy-Mechanism-of-RL#1]
First of all, thank you for your excellent work on this project. It's really impressive and helpful.

I have a question regarding the methods mentioned in the paper. You've provided the implementation for kl_cov, but I couldn't find the implementation for clip_cov. Would it be possible to share the clip_cov implementation as well?

I've noticed that in the on-policy case, kl_cov doesn't actually take effect because the kl_penalty (log_prob - old_log_prob).abs() is always 0. However, clip_cov should still be effective in the on-policy scenario, as the pseudo-code in the paper shows a detach operation.

Given this, I'm interested in testing both methods in an on-policy setting, but it seems I can only use clip_cov for this purpose. I'm wondering if there's a specific reason why clip_cov isn't provided or recommended in practice. Is it because clip_cov has more hyperparameters and might be trickier to tune?

It would be great to get some insights on this and possibly see the clip_cov implementation. This would allow for a more comprehensive comparison of both methods in various scenarios, including on-policy cases. Thank you for your time and consideration.

**Comment by JC-Chen1** [comment]
Implementation of `clip_cov` has been released in https://github.com/PRIME-RL/Entropy-Mechanism-of-RL/commit/8bef19b10fc248b12983ab968c722cd6b1a4c6d1. Please feel free to use it and welcome for any feedback or discussion!

For tuning guildance of `clip_cov`, some critical settings might be helpful,

1. `clip_cov_ratio` defines the select ratio in `clip_cov`, usaually a higher `clip_cov_ratio` will trigger a stronger reaction on policy entropy.
https://github.com/PRIME-RL/Entropy-Mechanism-of-RL/blob/d6152e8aba4edd4bee4b69adcdc5fb844a7f3c85/verl/trainer/config/ppo_trainer.yaml#L54

2. `clip_cov_lb` and `clip_cov_ub` control the select range of high covariance. 
https://github.com/PRIME-RL/Entropy-Mechanism-of-RL/blob/d6152e8aba4edd4bee4b69adcdc5fb844a7f3c85/verl/trainer/config/ppo_trainer.yaml#L55-L56

For your reference, we have also released two training scripts (offline settings) for [7B](https://github.com/PRIME-RL/Entropy-Mechanism-of-RL/blob/main/recipe/dapo/7b_clip_cov.sh) and [32B](https://github.com/PRIME-RL/Entropy-Mechanism-of-RL/blob/main/recipe/dapo/32b_clip_cov.sh) models when using `clip_cov` method.
